Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -457,6 +460,49 @@ DasharoSystemFeaturesUiLibConstructor (
         );
     ASSERT_EFI_ERROR (Status);
   }
+
+  BufferSize = sizeof (mDasharoSystemFeaturesPrivate.DasharoFeaturesData.DmaProtection);
+  Status = gRT->GetVariable (
+      mDmaProtectionEfiVar,
+      &gDasharoSystemFeaturesGuid,
+      NULL,
+      &BufferSize,
+      &mDasharoSystemFeaturesPrivate.DasharoFeaturesData.DmaProtection
+      );
+
+  if (Status == EFI_NOT_FOUND) {
+    Status = gRT->SetVariable (
+        mDmaProtectionEfiVar,

```

Comment:
This variable is not used anywhere in edk2. SO still the DMA protection will work in VTD DXE driver

Reply:
it shoudl be used to set or clear bit0 in the VTD PCD

Reply:
> it should be used to set or clear bit0 in the VTD PCD
Added that to `DasharoBootPolicies/BootPolicies.c` with the rest.